packages/wow-ui/src/components/SwitchGroup/SwitchGroup.stories.tsx

```diff
@@ -0,0 +1,95 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { useState } from "react";
+
+import Switch from "@/components/Switch";
+import SwitchGroup from "@/components/SwitchGroup";
+
+const meta = {
+  title: "UI/SwitchGroup",
+  component: SwitchGroup,
+  tags: ["autodocs"],
+  parameters: {
+    componentSubtitle: "스위치 그룹 컴포넌트",
+  },
+  argTypes: {
+    children: {
+      description: "렌더링할 자식 요소를 나타냅니다.",
+      table: {
+        type: { summary: "ReactNode" },
+        defaultValue: { summary: null },
+      },
+      control: {
+        type: "function",
+      },
+      type: {
+        name: "function",
+        required: true,
+      },
+    },
+    value: {
+      description: "value는 외부에서 제어할 활성 상태를 나타냅니다.",
+      table: {
+        type: { summary: "boolean[]" },
+        defaultValue: { summary: null },
+      },
+      control: {
+        type: "array",
+      },
+    },
+    onChange: {
+      description: "외부 활성 상태가 변경될 때 호출될 콜백 함수를 나타냅니다.",
+      table: {
+        type: { summary: "(index: number) => void" },
+        defaultValue: { summary: null },
+      },
+      control: {
+        type: "function",
+      },
+    },
+  },
+} satisfies Meta<typeof SwitchGroup>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const Primary: Story = {
+  args: {
+    children: (
+      <>
+        <Switch text="Switch 1" />
+        <Switch isDisabled text="Switch 2" />
+        <Switch text="Switch 3" />
+      </>
```

Comment:
<img width="976" alt="image" src="https://github.com/GDSC-Hongik/wow-design-system/assets/89445100/cafb6f79-6d55-43ac-ba5f-b647dd32fdb4">


p2;
`children` 관련해서 타입 에러가 발생합니다.
```suggestion
    children: [
      <Switch text="Switch 1" />,
      <Switch isDisabled text="Switch 2" />,
      <Switch text="Switch 3" />,
    ],
```